src/test/resources/com/znsio/teswiz/features/mobile-emulation-on-web-browser.feature

```diff
@@ -7,4 +7,10 @@ Feature: Mobile Emulation on Web browser
     Given "someone" starts "images-web" on "chrome-mobile1"
     And "someone-else" starts "bing-web" on "firefox-tab2"
     When "someone" searches for "bear"
-    Then "someone-else" searches for "tiger"
\ No newline at end of file
+    Then "someone-else" searches for "tiger"
+
+    #  CONFIG=./configs/android_emulator_config.properties TAG="@mobileEmulation and @android-chrome" ./gradlew run
+  @android-chrome @android
+  Scenario: Sample test to launch chrome on android emulator
+    Given "I" starts "chrome-android" on "chrome"
```

Comment:
It looks very odd to me
you can write something like
Given: I launch chrome browser on my android emulator